### DISCUSSION TITLE
zh-cn/posts/2024/11/02/physical-logical-pixel-resolution/

### DISCUSSION THREAD

**Discussion by giscus** [CyrusYip/blog-comments#81] [Comments]
# zh-cn/posts/2024/11/02/physical-logical-pixel-resolution/

物理（physical）表示实际的值，逻辑（logical）表示经过缩放的值。
物理像素 # 像素（pixel）是显示器的基本单位，一个像素显示一种颜色。为了和逻辑像素（logical pixel）区分，像素（pixel）又称物理像素（physical pixel）。

https://cyrusyip.org/zh-cn/posts/2024/11/02/physical-logical-pixel-resolution/

**Comment by laozidabo** [comment]
学到了 谢谢你

**Comment by Liam-Zhong** [comment]
>Physical resolution
>2561.4001017808914×1600.200063586235
>Logical resolution
>1423×889
>Viewport size
>1328×773
>Scale
>180.00000715255737%

请问视窗大小是什么嘞

**Reply by CyrusYip** [reply]
视口（viewport）就是浏览器显示内容的区域，不包括地址栏这些不用于显示内容的区域。浏览器全屏的时候视口大小就和逻辑分辨率一样了。你也可以调整浏览器窗口大小，感受一下视口大小的变化。

**Reply by Liam-Zhong** [reply]
噢哦其实我大概能猜出来，但我就是全屏显示结果还不一样，想了下估计和开了字体缩放 120% 有关系。谢谢博主解答！
